devtools/common/src/test/java/io/quarkus/cli/commands/CreateProjectTest.java

```diff
@@ -243,6 +246,7 @@ void createMultipleTimes() throws InterruptedException {
         }).collect(Collectors.toList());
         executorService.invokeAll(collect);
         latch.await();
+        System.out.println(watch.toString());
```

Comment:
Hmmm, what's the purpose of this?

Reply:
It logs the test duration. I thought it was not a problem to leave it here. should I remove it?

Reply:
I would say that I wouldn't import a Google class just for that. So I would remove it for now and revisit if the changes you made are not stable enough.